core/domain/exp_domain.py

```diff
@@ -2236,6 +2236,15 @@ def _convert_states_v7_dict_to_v8_dict(cls, states_dict):
         return states_dict
 
     @classmethod
+    def _convert_states_v8_dict_to_v9_dict(cls, states_dict):
+        """Converts from version 8 to 9. Version 9 contains correct"""
```

Comment:
Perhaps this could be more descriptive? Something like "Version 9 contains 'correct' field in answer groups."